add-link

```diff
@@ -40,3 +40,5 @@ http://eternalstarculture.com/intern/home.php
 https://tinyurl.com/4cbf4wcp
 https://rachelleckman.com/.pdf
 https://condocopia.org/contact.html
+https://carlicence.co.za/wp-admin/shiplabel/
```

Comment:
Seems fixed to me

![image](https://user-images.githubusercontent.com/44526987/125175784-b0c2c880-e1ce-11eb-867b-82ae2a78a43b.png)